tool_collections/gatk/variant_combine/.shed.yml

```diff
@@ -1,4 +1,14 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/variant_combine
-owner: devteam
+homepage_url: https://www.broadinstitute.org/gatk/
+categories:
+- Sequence Analysis
```

Comment:
All GATK tools should have `Variant Analysis` associated and not Sequence Analysis.